release-checklist.md

```diff
@@ -0,0 +1,21 @@
+# Release checklist
+- [ ] `git flow release start v19.3.0` (`v19.3.0-beta.1` if beta)
+- [ ] Check that all new changes are reflected in the changelog
+- [ ] Bump package.json version to `v19.3.0` (`v19.3.0-beta.1` if beta)
+- [ ] `nvm ls` Make sure you are on node v10+ 
+- [ ] `npm run build`
+- [ ] `npm run test`, make sure all tests are passing
+- [ ] Commit changes
+- [ ] `npm publish` or `npm publish --tag beta` for beta
+- [ ] Enter OTP
+- [ ] Make sure your `master` and `develop` branches are up-to-date
+- [ ] `git flow release finish v19.3.0` (`v19.3.0-beta.1` if beta)
```

Comment:
- [ ] `git flow release finish VERSION_NUMBER` 
   For example, `19.3.0` or 19.3.0-beta.1` are both valid `VERSION_NUMBER` values.